Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

```diff
@@ -215,6 +215,18 @@ Instances deployed in AWS will sync CAM **Manual Terminate Protection** with Dis
 
 Instances imported from AWS will inherit DisableApiTermination flag into **Manual Terminate Protection** flag. Flag status is provided when provider is syncronized, so changes are not reflected in CAM until **Sync** operation has been done.
 
+#### Instance protection on Google Cloud Platform instances
```

Comment:
Please, use level 3 title and update ToC and metadata (date and add you as an author, at least):
```suggestion
### Instance protection on Google Cloud Platform instances
```